Assets/scripts/MelodyUnit.cs

```diff
@@ -0,0 +1,50 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Collections;
+
+public class MelodyUnit : Unit {
+    public Color LeadingColor;
+    public override void Initialize()
+    {
+        base.Initialize();
+        transform.position += new Vector3(0, 0, -1);
+        GetComponent<Renderer>().material.color = LeadingColor;
```

Comment:
Our units won't have renderers / materials since they're 2D sprites. Are we considering going 3D?